order/controller.go

```diff
@@ -198,14 +198,14 @@ func (c *Controller) processTrade(order *model.Order) {
 	// register order volume
 	c.Results[order.Pair].Volume += order.Price * order.Quantity
 
-	// calculate profit only for sell orders
+	// calculate profit only to sell orders
 	if order.Side != model.SideTypeSell {
 		return
 	}
 
 	profitValue, profit, err := c.calculateProfit(order)
 	if err != nil {
-		c.notifyError(fmt.Errorf("order/controller storage: %s", err))
```

Comment:
I also remove these errors replacements, it will help to pass the original error forward

Reply:
Yes, that's definitely better idea than creation of new error and forwarding the original. I was thinking about doing it, but to be honest I had not enough time to do it that way.